src/models/CommonModel.ts

```diff
@@ -49,26 +50,38 @@ export class CommonModel extends CommonSchema<CommonModel>{
                 });
             }
         }
+
+
+        const mergeItems = (models: CommonModel | CommonModel[] | undefined) : CommonModel | undefined => {
+            if(models !== undefined){
+                if(Array.isArray(models)){
+                    if(models.length > 0){
+                        let mergedItemsModel : CommonModel = models[0];
+                        models.forEach((model) => {mergedItemsModel = CommonModel.mergeCommonModels(mergedItemsModel, model, originalSchema);});
+                        return mergedItemsModel;
+                    } else {
+                        return undefined;
+                    }
+                } 
+            }
+            return models;
```

Comment:
You can simplify this by:

```ts
if(Array.isArray(models) && models.length > 0) {
  // logic
} else {
  return undefined;
}
```

Reply:
Cant use exactly that since if `models` is not array I want it to return `models` as is. But I simplified it to 
```
if(Array.isArray(models)){
       if(models.length > 0){
           ...
       } else {
              return undefined;
       }
}
...
```